src/Build/Definition/ProjectItem.cs

```diff
@@ -895,6 +863,49 @@ private ProjectMetadata GetItemDefinitionMetadata(string name)
             return metadataFromDefinition;
         }
 
+        /// <summary>
+        /// Returns the metadata with the specified key.
+        /// Returns null if returnNullIfNotFound is true otherwise returns empty string when metadata not present
+        /// </summary>
+        public string GetMetadataValueEscaped(string name, bool returnNullIfNotFound)
```

Comment:
Is it intentional to expose the method publicly? The existing overload is implemented explicitly, which keeps it internal.

```suggestion
        string IItem.GetMetadataValueEscaped(string name, bool returnNullIfNotFound)
```

Reply:
I think it‘s same with HasMetadata，SetMetadata and GetMetada， so make is public

Reply:
True, `HasMetadata`, `GetMetadataValue`, and `SetMetadataValue` are public.

But the existing `GetMetadataValueEscaped(string name)` is not, so unless there's a need to expose the new `GetMetadataValueEscaped(string name, bool returnNullIfNotFound)`, it should stay internal.

Reply:
OK. I will update that.